### PR TITLE
feat(mantine, table): create a new LastUpdated component

### PR DESCRIPTION
**Pull request by GermainBergeron**
### Proposed Changes

Create a new component that displays the last time the table updated. It hints the user when the table automatically refreshes

### Potential Breaking Changes

None, new component

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/mantine/src/components/table/TableLastUpdated.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     *
+     * @default "Last update:"
+     */
+    label?: string;
+    /**
+     * Array of dependencies that triggers an update
+     *
+     * @default []
+     */
+    dependencies?: any[];
```
Comment (gdostie) [inline packages/mantine/src/components/table/TableLastUpdated.tsx:30]: In what use cases do we need to pass this prop?

Comment (GermainBergeron) [reply packages/mantine/src/components/table/TableLastUpdated.tsx:30]: If your table is auto refreshed e.g, with an interval or when you select a facet that is outside the table you need to update the LastUpdated somehow

Comment (gdostie) [reply packages/mantine/src/components/table/TableLastUpdated.tsx:30]: Can we always pass `[data]`?

Comment (GermainBergeron) [reply packages/mantine/src/components/table/TableLastUpdated.tsx:30]: Hmm I was thinking that for a client side table it is not necessary, but it wouldn't hurt. I'm just wondering how we would do it. I'll try something with cloneElement

**Review by gdostie — APPROVED** [review]

**Pull request merged**